src/customers/account-dashboard-my-downloadable-products.md

```diff
@@ -19,3 +19,16 @@ _My Downloadable Products_
 
     ![]({% link images/images/account-dashboard-download-video.png %}){: .zoom}
     _Download Video_
+
+{:.bs-callout-info}
+To add a product to the My Downloadable Products list, customer need to create an order with a product of the **downloadable** type. See [Downloadable Product]({% link catalog/product-create-downloadable.md %})
+
+## Column description
+
+|Column|Description|
+|--- |--- |
+|Order#|The [order]({% link /sales/orders.md %}) in which the downloadable product was purchased.|
+|Date| Order creation date.|
+|Title|The name of the downloadable product that was purchased with the order.|
+|Status|Order processing status.|
+|Remaining Downloads|Number of available downloads of the downloaded product.|
```

Comment:
This looks as if you are describing the columns for the **My Downloadable Products** list. Placing this at the end disturbs the natural flow. I would place this just after the screenshot so that the reader can easily make the connection.

For the Title column, perhaps you can also explain that it contains the download link? Is that the case? If I am wrong, never mind.

```suggestion
The following table describes the _My Downloadable Products_ values:

|Column|Description|
|--- |--- |
|Order#|The [order]({% link /sales/orders.md %}) in which the downloadable product was purchased.|
|Date| Order creation date, which provides a link to the order detail.|
|Title|The name of the downloadable product purchased with the order. Provides a link to the downloadable product.|
|Status|Order processing status.|
|Remaining Downloads|Number of available downloads of the downloaded product.|
```

Reply:
If I am incorrect in the link descriptions, go ahead and change to the correct meaning. I was taking a guess. 😉

Reply:
Ohh, yes @hguthrie, you totally right!
Thank you for your great notice ☺️
![image](https://user-images.githubusercontent.com/78729171/116308037-b2741700-a7af-11eb-857b-cf83ef4c49e9.png)

This is done too, please approve!